lab1/syntax.txt

```diff
@@ -0,0 +1,22 @@
+program = “#”cmpdstmt “?”
+decllist = declaration | declaration “;” decllist
+declaration = ”let” IDENTIFIER “:” type
+simpletype = “bool” | “char” | “integer"
+arraytype = simpletype “[“ integer “]”
+type = simpletype | arraytype | "string"
+cmpdstmt = “{“ {stmt} “}”
```

Comment:
Those {} don't appear in your code sample. Consider removing them or updating your language.